_data/defs/key-events.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- attr: year 
+  type: number / string 
+  short: Year 
+  description: "Each event is given a year, range, or approximation (eg. 2011, 2001-2005, 1980s)"
+  required: true
+
+- attr: aca_year 
```

Comment:
`aca_year` why not just `academic_year`? I'm generally not a fan of obscure variable names as they make it harder to understand the code.

Reply:
Very valid!